zebra-state/src/service/non_finalized_state/chain.rs

```diff
@@ -6,13 +6,18 @@ use std::{
 
 use tracing::{debug_span, instrument, trace};
 use zebra_chain::{
-    block, orchard, primitives::Groth16Proof, sapling, sprout, transaction,
-    transaction::Transaction::*, transparent, work::difficulty::PartialCumulativeWork,
+    block::{self, ChainHistoryMmrRootHash},
+    history_tree::HistoryTree,
+    orchard,
+    primitives::Groth16Proof,
+    sapling, sprout, transaction,
+    transaction::Transaction::*,
+    transparent,
+    work::difficulty::PartialCumulativeWork,
 };
 
 use crate::{PreparedBlock, Utxo, ValidateContextError};
 
-#[derive(Default, Clone)]
```

Comment:
Does anything rely on `Chain::default()` ?

Reply:
There was never really a sensible default value here, because the chain was always forked off the finalized state.

So code that depends on a default is probably wrong.